Software/Python/easygopigo3.py

```diff
@@ -511,7 +511,7 @@ def read_encoders_average(self, units="cm"):
             # do no conversion
         return average
 
-    def turn_degrees(self, degrees, blocking=False):
+    def turn_degrees(self, degrees, blocking=True):
         """
         | Makes the `GoPiGo3`_ robot turn at a specific angle while staying in the same spot.
```

Comment:
Even if the documentation needs a major overhaul, we should still adjust it where things might get forgotten in the future. In this case, it's the `blocking` parameter.